### PR TITLE
Add note about System Lightning hooks

### PR DESCRIPTION
**Pull request by jonashaag**
Refs #411

### CODE REVIEW

File: asteroid/engine/system.py [hunk 1]
```diff
@@ -10,6 +10,13 @@ class System(pl.LightningModule):
     """Base class for deep learning systems.
     Contains a model, an optimizer, a loss function, training and validation
     dataloaders and learning rate scheduler.
+    
+    Note that by default, any PyTorch-Lightning hooks are *not* passed to the model.
+    If you want to use Lightning hooks, add the hooks to a subclass::
+    
+        class MySystem(System):
+            def on_train_batch_start(self, batch, batch_idx, dataloader_idx):
+                return self.model.on_train_batch_start(batch, batch_idx, dataloader_idx)
```
Comment (mpariente) [inline asteroid/engine/system.py:19]: Should this be a code block?

**Comment by mpariente** [comment]
Let's fix the linter and merge, thanks.

File: asteroid/engine/system.py (hunk 1 above)
Comment (jonashaag) [reply asteroid/engine/system.py:19]: It is, `::` at the end of a line is special ReST syntax for "insert : and expect a code block"

**Comment by jonashaag** [comment]
Did you install /lint now or not? :D

**Comment by mpariente** [comment]
I couldn't make it work.. 
If you want to give it a try, feel free

**Comment by mpariente** [comment]
The action did run, successfully, but it tells there is nothing to lint.. 

I'd have to dig into that a bit more. 
Maybe it's because we lost the info of the branch on the way, and it's linting master.
Really don't know. 

image

**Comment by jonashaag** [comment]
It’s 2021 and we’ll soon all be driven around in our autonomous cars but  auto linting and formatting GitHub PRs is an unsolved problem

**Pull request merged**